doc/source/user_guide/metadata_structure.rst

```diff
@@ -0,0 +1,361 @@
+
+cut_on_wavelength
+    type: float
+
+    Cut on wavelength in nm in case of a long-pass or bandpass filter.
+
+cut_off_wavelength
+    type: float
+
+    Cut off wavelength in nm in case of a short-pass or bandpass filter.
+
+Spectrometer
+============
+
+Contains information about the spectrometer, configuration and grating used
+for the measurement. In case multiple spectrometers are connected in series,
+they should be numbered `Spectrometer_1`, etc.
+
+model
+    type: string
+
+    Model of the spectrometer (branding by manufacturer).
+
+acquisition_mode
+    type: string
+
+    Acquisition mode (e.g. 'Parallel dispersive', versus 'Serial dispersive').
+
+entrance_slit_width
+    type: float
+
+    Width of the entrance slit in mm.
+
+exit_slit_width
+    type: float
+
+    Width of the exit slit (serial acquisition) in mm.
+
+central_wavelength
+    type: float
+
+    Central wavelength during acquisition (parallel acquisition).
+    
+start_wavelength
+    type: float
+
+    Start wavelength in nm (serial acquisition).
+
+step_size
+    type: float
+
+    Step size in nm (serial acquisition).
+
+Grating
+-------
+
+Information of the dispersion grating employed in the measurement.
+
+groove_density
+    type: int
+
+    Density of lines on the grating in grooves/mm.
+
+blazing_angle
+    type: int
+
+    Angle in degree (º) that the grating is blazed at.
+
+blazing_wavelength
+    type: int
+
+    Wavelength that the grating blaze is optimized for in nm.
+
+Filter
+-------
+
+Information about additional filters entered into the lightpath after the
+sample. In case multiple filters are used, they should be numbered
+`Filter_1`, etc. See :ref:`filter-label` above for details on items that
+may potentially be included.
+
+Detector
+========
+
+Contains information about the detector used to acquire the signal. Contained
+leaves will differ depending on the type of detector.
+
+detector_type
+    type: string
+
+    The type of detector used to acquire the signal (CCD, PMT, StreakCamera, 
+    TCSPD).
+
+model
+    type: string
+
+    The model of the used detector.
+
+frames
+    type: int
+
+    Number of frames that are summed to yield the total integration time.
+
+integration_time (s)
+    type: float
+
+    Time over which the signal is integrated. In case multiple frames are
+    summed, it is the total exposure time. In case of serial acquisition, it is
+    the dwell time per data point.
+
+saturation_fraction
+    type: float
+
+    Fraction of the signal intensity compared with the saturation threshold of
+    the CCD.
+
+binning
+    type: tuple of int
+
+    A tuple that describes the binning of a parallel detector such a CCD on
+    readout in x and y directions.
+
+processing
+    type: string
+
+    Information about automatic processing performed on the data, e.g. 'dark
+    subtracted'.
+
+sensor_roi
+    type: tuple of int
+
+    Tuple that specifies range of pixels on a detector that are read out.
```

Comment:
Specify the order of the values of the tuple? Which one is position, size, x, y, etc.?